documentation/source/user_section/tutorials/shape-analysis/compute-csa-and-other-shape-metrics/other-shape-metrics.rst

```diff
@@ -10,9 +10,9 @@ The csv files generated by ``sct_process_segmentation`` include besides the cros
 ================  =======
 ``diameter_AP``   Antero-posterior diameter (AP diameter)
 ``diameter_RL``   Transverse diameter
-``eccentricity``  Eccentricity
-``solidity``      Solidity
-``orientation``   Orientation
+``eccentricity``  Eccentricity of the ellipse that has the same second-moments as the spinal cord. Eccentricity is the ratio of the focal distance over the major axis length. The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle.
+``solidity``      Solidity. CSA(spinal_cord) / CSA_convex(spinal_cord). If perfect ellipse, it should be 1. This metric is interesting for detecting non-convex shape (e.g., in case of strong compression).
```

Comment:
Maybe we can consider removing `If perfect ellipse, it should be 1.`. It is not clear what a perfect ellipse is; see [here](https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/4#issuecomment-1758381528).

Reply:
Good catch! Definitely. :)

Reply:
I think what the quoted part is trying to say is:

- Solidity will be 1 when the area of the cord is equal to the area of the convex shape that bounds the cord.
- This is only true when the spinal cord is a perfect circle.
- So, the closer the cord is to 1, the closer the cord is shaped like a circle.

Maybe there's a better way to phrase this?

Reply:
Tweaked the wording in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4262/commits/d8da9eec8db9bc5597b5eb5cf4e3d47b42c1cba6 and https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4262/commits/15e364912da84b73c15712e91fb2e31374c8466c.